client/src/components/DisplayTable.js

```diff
@@ -78,6 +78,18 @@ export class DisplayTable extends React.Component {
       .tap( descending ? _.noop : _.reverse )
       .value();
 
+    const all_sort_keys = _.chain(rows)
+      .map( row => _.keys(row.sort_values) )
+      .flatten()
```

Comment:
Real minor, but could just be `flatMap` here and below.

Reply:
ah TIL :+1: